test/beast/zlib/deflate_stream.cpp

```diff
@@ -433,6 +433,7 @@ class deflate_stream_test : public beast::unit_test::suite
     {
         doMatrix(c, "Hello, world!", &self::doDeflate1_beast);
         doMatrix(c, "Hello, world!", &self::doDeflate2_beast);
+        log << "no-silence keepalive" << std::endl;
```

Comment:
oh no... why?

Reply:
The Travis build using valgrind times out after 10 minutes of silence
Since we now test both Beast.zlib and upstream zlib, the build exceeds the silence limit

Reply:
See [this Travis job](https://travis-ci.org/AeroStun/beast/jobs/601715039)